flink-connector-oracle-cdc/src/test/java/com/ververica/cdc/connectors/oracle/table/OracleConnectorITCase.java

```diff
@@ -321,6 +321,74 @@ public void testStartupFromLatestOffset() throws Exception {
         result.getJobClient().get().cancel().get();
     }
 
+    @Test
+    public void testConsumingNumericColumns() throws Exception {
+        // Prepare numeric type data
+        try (Connection connection = OracleTestUtils.testConnection(oracleContainer);
+                Statement statement = connection.createStatement()) {
+            statement.execute(
+                    "CREATE TABLE debezium.issue552 (ID NUMBER, WEIGHT NUMBER, PRICE FLOAT(63), PRIMARY KEY (ID))");
```

Comment:
How about `debezium.issue552` to `debezium.test_numeric_table` ？
and we can define more columns with number types which can be defined as `BOLEAN`, `TINYINT`...`DECIMAL` in Flink SQL DDL, we can cover all types in this way.

Reply:
Do you have time to update this @ashulin ?

Reply:
I'll update it tonight. @leonardBang